unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/AvatarModifierArea.cs

```diff
@@ -175,6 +187,12 @@ private void ApplyCurrentModel()
 
                 OnAvatarEnter += modifier.ApplyModifier;
                 OnAvatarExit += modifier.RemoveModifier;
+
+                string messageToAdd = modifier.GetWarningDescription();
+                if (!DataStore.i.HUDs.inAvatarModifierAreaWarningDescription.Contains(messageToAdd))
+                {
+                    DataStore.i.HUDs.inAvatarModifierAreaWarningDescription.Add(messageToAdd);
+                }
```

Comment:
Had to change it to a List<string> with the description of the AvatarModifier. Had to do it this way since there was a cyclic reference for DataStore_HUDs and AvatarShape

Reply:
You can just move `IAvatarModifier` into a new assembly, reference it wherever is needed and you wont have cyclic references because `IAvatarModifier` is a "leaf" assembly that (if so) would only reference other "leaf" assemblies.